packages/zowe-explorer/src/utils/SettingsConfig.ts

```diff
@@ -40,6 +47,20 @@ const currentVersionNumber = semver.major(
     vscode.extensions.getExtension("zowe.vscode-extension-for-zowe").packageJSON.version
 );
 
+async function promptReload() {
+    // Prompt user to reload VS Code window
+    const reloadButton = localize("standardization.reload.button", "Reload Window");
+    const infoMsg = localize(
+        "standardization.reload.infoMessage",
+        "Settings have been migrated successfully, please reload VS Code window."
```

Comment:
@rudyflores We might want to add more details to this message. Perhaps something like below:

`Settings have been successfully migrated for Zowe Explorer version 2 and above. To apply these settings, please reload your VS Code window.`